heroku/cli
# chore(cli): add output for waiting logs

## Description
[Work Item](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000023uOH0YAM/view)

This PR adds output to the `heroku logs` command to let the user know it is waiting for logs to be produced for Fir apps.

## Testing
1. Pull down branch & `yarn` it up
2. Ensure initial log output (`'Waiting for logs...'`) is visible when running logs via `./bin/run logs -a <fir-app>`
3. Open your fir app and refresh the url. Ensure logs populate after initial log output
